shiny/playwright/controller/_accordion.py

```diff
@@ -238,10 +238,7 @@ def expect_width(self, value: StyleValue, *, timeout: Timeout = None) -> None:
         _expect_style_to_have_value(self.loc_container, "width", value, timeout=timeout)
 
     def expect_open(
-        self,
-        value: list[PatternOrStr],
-        *,
-        timeout: Timeout = None,
+        self, value: list[PatternOrStr], *, timeout: Timeout = None, **kwargs
```

Comment:
Do not expose the parameter publicly

```suggestion
        self,
        value: list[PatternOrStr],
        *,
        timeout: Timeout = None,
```